cfgov/jobmanager/migrations/0010_cleanup_unused_fields.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yeah, I wasn't sure what a nice (yet not terribly long name) would be for these. I excluded `jobs` from the filenames because they're within the context of the `jobmanager` app, so I didn't think it added much.